semaphore/scss_lint.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 
-changed=$(git diff --name-only origin/master apps/site/assets/css)
+fork_point=$(git merge-base --fork-point origin/master)
+changed=$(git diff --name-only $fork_point "*.ex" "*.exs")
```

Comment:
I think you might have forgot to edit this one to be relevant to the CSS files rather than the Elixir ones?

Reply:
Copy-paste mishap 😱